vignettes/Spectra.Rmd

```diff
@@ -393,7 +393,17 @@ affecting the total number of spectra are:
   precursor charge(s).
 - `filterPrecursorScan`: retains (parent and children) scans of an acquisition
   number.
+- `filterRanges`: retains spectra within user defined `ranges` for one or more
+  available `spectraVariables` in object. if multiple ranges/spectra variables
```

Comment:
Please use `. If multiple ...` and maybe also update with the suggested change for the documentation above for MsBackend.R